sql/moz-fx-data-shared-prod/org_mozilla_ios_firefox_derived/baseline_clients_first_seen_v1/init.sql

```diff
@@ -12,7 +12,9 @@ AS
 WITH baseline AS (
   SELECT
     client_info.client_id,
-    sample_id,
+      -- Some Glean data from 2019 contains incorrect sample_id, so we
+      -- recalculate here; see bug 1707640
+    udf.safe_sample_id(client_info.client_id) AS sample_id,
```

Comment:
This is an additional unrelated change picked up from regenerating these example usage queries (which allows them to be used in tests). Looks like this one was out of date.